sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java

```diff
@@ -264,6 +297,12 @@ public CreateNewContactSteps(
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(50);
           webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(UUID_INPUT);
         });
+    When(
+        "^I click on SAVE new contact case button$",
+        () -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
```

Comment:
maybe addin here to wait after the confirmation popup to appear? or something to make it stable

Reply:
There are different possibilities of popups available after that Saving action and I did it without waite here on purpose